packages/web-react/src/components/Alert/index.html

```diff
@@ -0,0 +1 @@
+{{> demo }}
```

Comment:
This file will then be in all components?

Reply:
There will be `demo/index.tsx`.